templates/quiz/start-button.php

```diff
@@ -53,19 +56,54 @@
 						 * @param LLMS_Quiz   $quiz        The current quiz instance.
 						 * @param LLMS_Lesson $lesson      The parent lesson instance.
 						 */
-						echo apply_filters( 'lifterlms_begin_quiz_button_text', __( 'Start Quiz', 'lifterlms' ), $quiz, $lesson );
+						echo apply_filters( 'lifterlms_begin_quiz_button_text', $start_button_string, $quiz, $lesson );
 					?>
 				</button>
 
 			<?php else : ?>
 				<p><?php _e( 'You are not able to take this quiz', 'lifterlms' ); ?></p>
 			<?php endif; ?>
 
-			<?php if ( $lesson->get_next_lesson() && llms_is_complete( get_current_user_id(), $lesson->get( 'id' ), 'lesson' ) ) : ?>
-				<a href="<?php echo get_permalink( $lesson->get_next_lesson() ); ?>" class="button llms-button-secondary llms-next-lesson"><?php _e( 'Next Lesson', 'lifterlms' ); ?></a>
-			<?php endif; ?>
 		</form>
 
+		<?php if ( $quiz->can_be_resumed() && $quiz->can_be_resumed_by_student() ) : ?>
+
+			<form method="POST" action="" name="llms_resume_quiz" enctype="multipart/form-data">
+
+				<?php if ( $quiz->is_open() ) : ?>
```

Comment:
mmm what if the quiz is closed, meaning the you reached the maximum number of attempts? you can't resume the latest incomplete attempt that was set to be resumed?

Reply:
Adjusted the if statement now.